ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -276,3 +276,57 @@ def gather(
         "batch_dims": batch_dims
     }
     return _get_node_factory_opset8().create("Gather", inputs, attributes)
+
+
+@nameable_op
+def max_pool(
+    data: NodeInput,
+    strides: List[int],
+    dilations: List[int],
+    pads_begin: List[int],
+    pads_end: List[int],
+    kernel_shape: TensorShape,
+    rounding_type: str = "floor",
+    auto_pad: Optional[str] = None,
+    index_element_type: Optional[str] = "i64",
+    axis: Optional[int] = 0,
+    name: Optional[str] = None,
+) -> Node:
+    """Perform max pooling operation and return both values and indices of the selected elements.
+
+    @param  data:                The node providing input data.
+    @param  strides:             The distance (in pixels) to slide the filter on the feature map
+                                 over the axes.
+    @param  dilations:           The dilation of filter elements(distance between elements).
+    @param  pads_begin:          The number of pixels to add at the beginning along each axis.
+    @param  pads_end:            The number of pixels to add at the end along each axis.
+    @param  kernel_shape:        The pooling operation kernel shape.
+    @param  rounding_type:       Determines used rounding schema when computing output shape.
+                                 Acceptable values are: ['floor', 'ceil']
+    @param  auto_pad:            Determines how the padding is calculated. Acceptable values:
+                                 [None, 'same_upper', 'same_lower', 'valid']
+    @param  index_element_type:  The data type used for the indices outputof this operator.
```

Comment:
You can add a note of default value `i64`. Moreover should it be `output of` with a space between words?